docs/src/cookbooks/cookbook.md

```diff
@@ -62,26 +64,35 @@ class Foo extends RawModule {
 
 Use the [`asTypeOf`](https://www.chisel-lang.org/api/latest/chisel3/UInt.html#asTypeOf[T%3C:chisel3.Data](that:T):T) method to reinterpret the [`UInt`](https://www.chisel-lang.org/api/latest/chisel3/UInt.html) as the type of the [`Bundle`](https://www.chisel-lang.org/api/latest/chisel3/Bundle.html).
 
+If you have a bundle with many elements in a module interface,
```

Comment:
I am not sure what you are trying to do in this PR. You replaced an example that went with the text above, `"Use the .asTyepeOf"` with a different example that is showing your different point. Why did you delete the example that was there before?